koku/api/provider/models.py

```diff
@@ -198,7 +198,7 @@ def delete(self, *args, **kwargs):
             with schema_context(self.customer.schema_name):
                 LOG.info(f"PROVIDER {self.name} ({self.pk}) CASCADE DELETE -- SCHEMA {self.customer.schema_name}")
                 _type = self.type.lower()
-                self._normalized_type = _type[: _type.index("-")] if _type.endswith("-local") else _type
+                self._normalized_type = _type.removesuffix("-local")
```

Comment:
Even better! I didn't know about that method.

Reply:
`removesuffix` and `removeprefix` are new in Python 3.9!